org.eclipse.wb.core/src/org/eclipse/wb/internal/core/utils/ui/SwtResourceManager.java

```diff
@@ -132,12 +131,7 @@ public static Image getImage(final int imageID) {
     image[0] = m_systemImageMap.get(imageID);
     if (image[0] == null) {
       final Display display = DesignerPlugin.getStandardDisplay();
-      display.syncExec(new Runnable() {
-        @Override
-        public void run() {
-          image[0] = display.getSystemImage(imageID);
-        }
-      });
+      display.syncExec(() -> image[0] = display.getSystemImage(imageID));

```

Comment:
Done by cleanup action but I still merge this one as it is a straight forward chagne.